src/main/kotlin/Graph.kt

```diff
@@ -0,0 +1,59 @@
+import Graph.Companion.STOCKHOLM
+import Graph.Companion.UPPSALA
+
+class Graph {
+    companion object {
+        const val STOCKHOLM = "ID_STOCKHOLM"
+        const val UPPSALA = "ID_UPPSALA"
+    }
+
+    val user: User? = null
+
+    fun findShortestPath(node1: Node, node2: Node): Int {
+        return 0
+    }
+
+    fun getNodeById(id: String): Node {
+
+        // TODO: ?
+        val firstName = user!!.firstName
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/628f49d8ab250ce000d5c68bd90683ae4b10e4e9/src/main/kotlin/Graph.kt#L19"></span>Calling !! on a nullable type will throw a NullPointerException at runtime in case the value is null. It should be avoided.</td>
    </tr>
  </tbody>
</table>